src/App.js

```diff
@@ -10,8 +10,10 @@ import QueueDisplay from "./QueueDisplay.js";
 import BuildHistoryDisplay from "./BuildHistoryDisplay.js";
 import GitHubStatusDisplay from "./GitHubStatusDisplay.js";
 import PerfHistoryDisplay from "./PerfHistoryDisplay.js";
+import PrDisplay from "./PrDisplay.js";
 import JobCorrelationHeatmap from "./JobCorrelationHeatmap.js";
 import GitHubActionsDisplay from "./GitHubActionsDisplay.js";
+import AuthorizeGitHub from "./GithubAuthorize.js";
```

Comment:
tiny nit but maybe rename to GitHubAuthorize.js for consistency?
```suggestion
import AuthorizeGitHub from "./GitHubAuthorize.js";
```